gcc4mbed/external/mbed/libraries/mbed/targets/hal/TARGET_NXP/TARGET_LPC43XX/PeripheralNames.h

```diff
@@ -12,6 +12,9 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
+ * CHANGE LOG:
+ * 10/8/2016 marcosfg - added pin names for adc only pins
```

Comment:
git is pretty good at keeping track of such things. Do such logs need to be added to every source file touched?